init/cli/logging.go

```diff
@@ -72,13 +82,28 @@ func (f *FileLoggers) SubProcessLogger(name string) launchlib.CreateLogger {
 }
 
 func (f *FileLoggers) OpenFile(path string) (*os.File, error) {
+	if _, ok := f.rotated[path]; !ok {
+		if _, ok := f.flags.(*truncatingFirst); ok {
+			rotate(path)
+			f.rotated[path] = true
+		}
+	}
 	file, err := os.OpenFile(path, f.flags.Get(path), f.mode)
 	if err != nil {
 		return file, errors.Wrapf(err, "could not open logging file '%s'", path)
 	}
 	return file, nil
 }
 
+func rotate(path string) {
+	limit := 5
```

Comment:
nit: explicitly declare this a const i.e. `const limit = 5`

Reply:
Done.